### PR TITLE
[fix] Updating README.md and TUTORIAL.md documentation

### PR DESCRIPTION
**Pull request by shedninja6**
### Summary
Updating the `README.md` documentation to match the new `yml` file type and finishing uncompleted sections in `TUTORIAL.md` so that fogg onboarding is smoother.

### Test Plan
No tests

### References
current README.md documentation as reference

### CODE REVIEW

File: docs/TUTORIAL.md [hunk 1]
```diff
@@ -92,60 +94,84 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     As we said before fogg works by generating code (terraform, make and bash) and the general workflow is–
 
-    1. edit `fogg.json`
+    1. edit `fogg.yml` 
+        - Add an `account_id` to the aws provider
+        ```shell
```
Comment (ryanking) [inline docs/TUTORIAL.md]:
```suggestion
        ```yaml
```

File: docs/TUTORIAL.md [hunk 2]
```diff
@@ -92,60 +94,84 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     As we said before fogg works by generating code (terraform, make and bash) and the general workflow is–
 
-    1. edit `fogg.json`
+    1. edit `fogg.yml` 
+        - Add an `account_id` to the aws provider
```
Comment (redhoteggplant) [inline docs/TUTORIAL.md]: is this just an example of an edit (and if so is it useful)? or does a user always have to do this?

File: docs/TUTORIAL.md [hunk 3]
```diff
@@ -164,53 +190,61 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     To create a new env, edit your fogg.json to look like this–
```
Comment (redhoteggplant) [inline docs/TUTORIAL.md]:
```suggestion
    To create a new env, edit your fogg.yaml to look like this–
```

File: docs/TUTORIAL.md [hunk 4]
```diff
@@ -287,33 +325,40 @@ Note that fogg works by generating Terraform and Make files. It does not run any
     .
     ├── Makefile
     ├── README.md
-    ├── fogg.json
+    ├── fogg.yml
+    ├── fogg.yml~
```
Comment (redhoteggplant) [inline docs/TUTORIAL.md]: what's this second file about actually 😮

Comment (shedninja6) [reply docs/TUTORIAL.md]: It's just a backup file that is automatically generated. It's harmless but not sure if it needs to be deleted or not. Thoughts @ryanking

Comment (ryanking) [reply docs/TUTORIAL.md]: That is probably created by your editor, not fogg, so you should probably leave it out of the listing.

File: docs/TUTORIAL.md (hunk 2 above)
Comment (shedninja6) [reply docs/TUTORIAL.md]: They always have to add an account id

**Pull request merged**